price_matching_system/templates/layout.html

```diff
@@ -32,26 +32,28 @@
                 </ul>
 
                 {% if g.user %}
+
                     <ul class="navbar-nav">
                         <li class="nav-item dropdown">
                             <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                                 <i class="bi bi-person-circle"></i> {{ g.user['UserName'] }}
                             </a>
                             <ul class="dropdown-menu dropdown-menu-end">
-                                <li><a class="dropdown-item" href="#">My Profile</a></li>
-                                <li><a class="dropdown-item" href="#">Change Details</a></li>
+                                <li><a class="dropdown-item" href="/user">My Profile</a></li>
+                                <li><a class="dropdown-item" href="/userEdit">Change Details</a></li>
```

Comment:
can we name the route differently? like /user/edit

Reply:
if for popular way, able to be changed, sure.

Reply:
changed to /user/edit